### ISSUE TITLE
retinaface使用FGD精度下降

### ISSUE THREAD

**Issue by 408550969** [yzd-v/FGD#42]
你好，我用retinaface训练自己的数据集，发现使用fgd后精度比不用低。teacher模型已验证精度，是正常的。我的teacher模型backbone是resnet50，student的backbone是mobilenetv3-small。
我将ssh后的特征层拿来计算fgd的loss，具体代码如下：
        out,features = net(images)
        loss_l, loss_c = criterion(out, priors, targets)
        loss = cfg['loc_weight'] * loss_l + loss_c
        with torch.no_grad():
            teacher_out,teacher_features = teacher_net(images)
        for i in range(3):
             ComputeFeatureLoss = FeatureLoss(features[i].shape[1],teacher_features[i].shape[1])
             ComputeFeatureLoss = ComputeFeatureLoss.cuda()
             distilloss=ComputeFeatureLoss(features[i],teacher_features[i],targets)
             loss= loss + distilloss
retinaface的targets是归一化的值，不是像素值，所以我去掉了img_metas。FeatureLoss的参数采用的默认值，请问造成精度较低的原因可能有哪些？

**Comment by yzd-v** [comment]
你可以检查下是否蒸馏环境下正确load了老师，以及global distillation的可学习参数部分是否参与了梯度更新，还有在蒸馏环境下不用distillloss能否和baseline保持一致。正确蒸馏下，同epoch表现会变好。
